### ISSUE TITLE
Test failure(s): binding.hue: fails sometimes

### ISSUE THREAD

**Issue by maggu2810** [eclipse/smarthome#1511]
https://hudson.eclipse.org/smarthome/job/SmartHomeDistribution-Nightly/1042/consoleText

```text
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.909 sec - in org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest
assert no discovery result for unknown device(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.378 sec
assert correct thing UID(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.009 sec
assert no thing UID for unknown device(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.001 sec
assert correct supported types(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.012 sec
assert valid DiscoveryResult(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.019 sec
Running org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest
Tests run: 31, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 40.399 sec - in org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest
assert command for brightness channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.511 sec
assert command for color channel: black(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.18 sec
assert command for color channel: white(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.157 sec
assert command for color channel: increase(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 3.375 sec
assert command for osram par16 50 for color temperature channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.126 sec
assert command for color channel: decrease(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.232 sec
assert command for osram par16 50 for brightness channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.115 sec
assert that HueLightHandler is registered and unregistered(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.09 sec
assert command for osram par16 50 for color temperature channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.106 sec
assert percentage value of color temperature when ct: 153(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.03 sec
assert percentage value of color temperature when ct: 326(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.014 sec
assert percentage value of color temperature when ct: 500(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.014 sec
assert command for color temperature channel: 0%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.108 sec
assert command for color temperature channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.103 sec
assert command for color channel: 0%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.102 sec
assert command for color channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.12 sec
assert command for brightness channel: 50%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.105 sec
assert command for brightness channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.099 sec
assert command for color temperature channel: 50%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.108 sec
assert command for color temperature channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.103 sec
assert command for color channel: 50%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.097 sec
assert command for color channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.103 sec
assert command for color channel: red(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.099 sec
assert command for brightness channel: increase(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 3.26 sec
assert command for alert channel(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 3.268 sec
assert command for brightness channel: decrease(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.179 sec
assert command for color temperature channel: 100%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.092 sec
assert command for osram par16 50 for brightness channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.092 sec
assert command for color channel: 100%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.092 sec
assert command for color channel: blue(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.094 sec
assert command for effect channel(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.182 sec
Running org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.19 sec - in org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest
assert that HueBridgeHandler is registered and unregistered(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.031 sec
assert that an existing user is used if authentication was successful(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.031 sec
assert correct status if authentication failed for old user(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.04 sec
verify status if link button is not pressed(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.027 sec
assert that a new user is added to config if not existing yet(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.025 sec
verify status if new user cannot be created(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.025 sec
Running org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.082 sec <<< FAILURE! - in org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest
assert hue light registration(org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest)  Time elapsed: 0.056 sec
assert startSearch is called(org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest)  Time elapsed: 0.011 sec  <<< FAILURE!
java.lang.AssertionError: 
Expected: is null
     but: was <org.eclipse.smarthome.binding.hue.handler.HueBridgeHandler@7f188fd>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at org.junit.Assert$assertThat.callStatic(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:53)
	at org.junit.Assert$assertThat.callStatic(Unknown Source)
	at org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest.setUp(HueLightDiscoveryServiceOSGITest.groovy:68)


Results :

Failed tests: 
  HueLightDiscoveryServiceOSGITest.setUp:68 
Expected: is null
     but: was <org.eclipse.smarthome.binding.hue.handler.HueBridgeHandler@7f188fd>

Tests run: 44, Failures: 1, Errors: 0, Skipped: 0
```

**Comment by afuechsel** [comment]
should be fixed with #1799. @maggu2810 please recheck.

**Comment by maggu2810** [comment]
Will close this issue until I run again into this failure.
